releases/2022-10-28-releasenotes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'We're .COMical and Always Hardening'
+description: 'Vendia is on .com, advanced Azure beta features, expanded success rate for a variety of transaction scenarios, updated SDK defaults, and enhanced experience within GraphQL Explorer'
+date: '2022-10-28'
+draft: false
+tags:
+- Improvements
+- Azure
+- Developer Experience
+---
+
+They're at it again! The Vendia Engineering team has a series of new improvements to continue advancing the experience building on Vendia.  
+
+As always, see our [Docs](https://www.vendia.com/docs/share) for the complete details of each new feature and our [Examples](https://github.com/vendia/examples) for more guidance. 
```

Comment:
```suggestion
As always, see [our docs](https://www.vendia.com/docs/share) for the complete details of each new feature and [our examples](https://github.com/vendia/examples) for more guidance. 
```

Reply:
Take or leave...

Reply:
I updated it in local branch and pushed - should be incorporated now